jupytext/contentsmanager.py

```diff
@@ -477,6 +478,10 @@ def get_config_file(self, directory):
                 if self.file_exists(path):
                     return path
 
+            pyproject_path = directory + "/" + PYPROJECT_FILE
```

Comment:
This is the location where I think we should test whether or not there is a jupytext entry in the pyproject file.
(BTW the `+ "/"` is correct, we could add a comment above: paths in Jupyter's contents manager always use the linux path separator, even on Windows)